README.md

```diff
@@ -79,87 +79,146 @@ make helm-chart-push
+
+### Deploy Fybrik module
+
+Deploy `FybrikModule` in `fybrik-system` namespace:
 ```bash
-kubectl get fybrikmodule hello-world-read-module -n fybrik-system
-kubectl describe fybrikmodule hello-world-read-module -n fybrik-system
+kubectl create -f hello-world-read-module.yaml -n fybrik-system
 ```
+### Test using Fybrik Notebook sample
+
+Execute the sections in [Fybrik Notebook sample](https://fybrik.io/v0.5/samples/notebook/) until `Register the dataset in a data catalog` section (excluded).
 
 
 ## Register data asset in a data catalog
 
 You need to register your data asset in a data catalog in order for it to be used by the `fybrik-manager`.
 
-- Follow step `Register the dataset in a data catalog` in [this example](https://fybrik.io/dev/samples/notebook/). These steps register the credentials required for accessing the dataset, and then register the data asset in the catalog.
+- Follow step `Register the dataset in a data catalog` in [this example](https://fybrik.io/v0.5/samples/notebook/). These steps register the credentials required for accessing the dataset, and then register the data asset in the catalog.
 
 - As an example you can run these commands to register two assets exist in `sample_assets`:
 ```bash
-kubectl apply -f sample_assets/assetMedals.yaml
-kubectl apply -f sample_assets/secretMedals.yaml
-kubectl apply -f sample_assets/assetBank.yaml
-kubectl apply -f sample_assets/secretBank.yaml
+kubectl apply -f sample_assets/assetMedals.yaml -n fybrik-notebook-sample
+kubectl apply -f sample_assets/secretMedals.yaml -n fybrik-notebook-sample
+kubectl apply -f sample_assets/assetBank.yaml -n fybrik-notebook-sample
+kubectl apply -f sample_assets/secretBank.yaml -n fybrik-notebook-sample
 ```
 
-## Define policies
+### Define data access policies
+
+  Define the following [OpenPolicyAgent](https://www.openpolicyagent.org/) policy to allow the write operation:
+
+```bash
+package dataapi.authz
+
+rule[{"action": {"name":"RedactAction", "columns": column_names}, "policy": description}] {
+  description := "Redact columns tagged as PII in datasets tagged with finance = true"
+  input.action.actionType == "read"
+  input.resource.tags.finance
+  column_names := [input.resource.columns[i].name | input.resource.columns[i].tags.PII]
+  count(column_names) > 0
+}
+
+rule[{"action": {"name":"RedactAction", "columns": column_names}, "policy": description}] {
+  description := "Redact columns tagged as sensitive in datasets tagged with finance = true"
+  input.action.actionType == "read"
+  input.resource.tags.finance
+  column_names := [input.resource.columns[i].name | input.resource.columns[i].tags.sensitive]
+  count(column_names) > 0
+}
+```
 
-You can define OpenPolicyAgent policy to apply them to datasets. You can follow the `Define data access policies` section in [this example](https://fybrik.io/dev/samples/notebook/).
+  In this sample only the policy above is applied. Copy the policy to a file named sample-policy.rego and then run:
```

Comment:
why do you need the sentence "In this sample only the policy above is applied." ?

Reply:
I removed it... thanks (its taken from Fybrik site btw)